docs/windows-service-instrumentation.md

```diff
@@ -0,0 +1,48 @@
+# Instrument a Windows Service running a .NET application
+
+We recommend setting environment variables for a Windows Service
+using the Windows Registry.
+
+> Remember to restart the Windows Service
+  after making changes to its configuration.
+
+## Set environment variables for a Windows Service
+
+The registry key of a given Windows Service (named `$svcName`) is located under:
+
+```powershell
+HKLM\SYSTEM\CurrentControlSet\Services\$svcName
+```
+
+The environment variables can be defined
+in a `REG_MULTI_SZ` (multiline registry value) called `Environment`
+in the following format:
+
+```env
+Var1=Value1
+Var2=Value2
+```
+
+Below is an example how instrumentation can be configured in PowerShell:
+
+```powershell
+$installationLocation = "C:\some\dir" # The path where you put the OTel .NET AutoInstrumentation binaries
```

Comment:
Do we have any recommended path for installation folder?

Reply:
No.